### PR TITLE
[IOTDB-4130] Implement sync module in the sender in the ASTVisitor to IConfigTaskExecutor section and development of SHOW PIPESINKTYPE 

### PR DESCRIPTION
**Pull request by 23931017wu**
### 1、Task
ShowPipeSinkTask
showPipeTask
CreatePipeSinkTask
CreatePipeTask
DropPipeSinkTask
DropPipeTask
StartPipeTask
StopPipeTask

## AFTER
图片

图片

### CODE REVIEW

**Comment by neuyilan** [comment]
@Cpaulyz please take a review~

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/sys/DropPipeSinkTask.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import org.apache.iotdb.db.mpp.plan.execution.config.ConfigTaskResult;
+import org.apache.iotdb.db.mpp.plan.execution.config.IConfigTask;
+import org.apache.iotdb.db.mpp.plan.execution.config.executor.IConfigTaskExecutor;
+import org.apache.iotdb.db.mpp.plan.statement.sys.DropPipeSinkStatement;
+
+import com.google.common.util.concurrent.ListenableFuture;
+
+public class DropPipeSinkTask implements IConfigTask {
+
```
Comment (neuyilan) [inline server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/sys/DropPipeSinkTask.java:30]: I think we can move the Pipe task to `server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/sys/pipe `package?

**Review by neuyilan — APPROVED** [review]
LGTM

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/sys/DropPipeSinkTask.java (hunk 1 above)
Comment (Cpaulyz) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/sys/DropPipeSinkTask.java:30]: Agree. We can move Task and Statement about sync to `org.apache.....sys.sync`

**Review by Cpaulyz — APPROVED** [review]
LGTM

File: server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/sys/DropPipeSinkTask.java (hunk 1 above)
Comment (23931017wu) [reply server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/sys/DropPipeSinkTask.java:30]: OK ，wait a second ！

**Pull request merged**